arwen/hosttest/execution_test.go

```diff
@@ -1452,8 +1405,7 @@ func TestExecution_ExecuteOnDestContext_Successful(t *testing.T) {
 			WithGasProvided(test.GasProvided).
 			Build()).
 		AndAssertResults(func(host arwen.VMHost, stubBlockchainHook *contextmock.BlockchainHookStub, verify *test.VMOutputVerifier) {
-			verify.
-				Ok().
+			verify.Ok().
 				// test.ParentAddress
```

Comment:
redundant comment? (multiple occurrences in this file)

Reply:
Deleted.